src/config/SettingsConfig.cc

```diff
@@ -36,6 +37,15 @@ struct SettingsParser : rapidsax::NullHandler
 	std::vector<Setting> settings;
 	std::vector<HotKey::Data> binds;
 	std::vector<std::string_view> unbinds;
+
+	struct ShortcutItem {
+		ShortcutIndex index = ShortcutIndex::NUM_SHORTCUTS;
+		ImGuiKeyChord keyChord = ImGuiKey_None;
+		bool repeat = false;
+		bool global = false;
+	};
```

Comment:
Thanks a lot for so quickly addressing many of my review comments.
BTW feel free to disagree and/or discuss my comments. Now it sometimes feels like I'm only dictating how I want the code to be, and you have no choice but to obey. You absolutely can disagree ;-)

One detail about this change. Why not a struct like:
```
struct ShortcutItem {
        ShortcutIndex index = NUM_SHORTCUTS;
        Shortcuts::Data data = {};
};
``` 
(And we have to change `Shortcuts::Data` to initialize the `keyChord` field, but that's  a good idea anyway.)
This ensures that this struct keeps in sync with possible future changes in `Shortcuts::Data`.

And we should probably also introduce a name `ShortcutIndex::INVALID`. It could have the same numerical value as `NUM_SHORTCUTS`. But using the same name to both indicate the number of indices and also an invalid index makes the code harder to read.